news_items/040_manual_grading_ui/info.json

```diff
@@ -0,0 +1,6 @@
+{
+    "uuid": "9a5c6fab-729d-4544-b6ac-98372fc34d25",
+    "title": "Manual Grading: New User Interface and Points Allocation",
+    "author": "<a href=\"https://github.com/mwest1066\">Matt West</a>",
```

Comment:
@mwest1066 I listed you as author based on previous discussion of having news items be authored by someone else, let me know if you'd like me to replace with someone else instead.